src/Wrapper/SeparatedVariation.php

```diff
@@ -36,6 +37,15 @@ public function isImageAvailable(array $imageAttributeValues, array $variationAt
         foreach ($imageAttributeValues as $imageAttributeValue) {
             foreach ($variationAttributes as $variationAttribute) {
                 $attributeData = explode(self::ATTRIBUTE_VALUE_SEPARATOR, $variationAttribute);
+
+                /**
+                 * if variation has only one groupable attribute, it's key value will be main variation id.
+                 * For this reason $attributeData will not contain an attribute id and attribute value id.
+                 */
```

Comment:
Even for comments over multiple lines, always use inline comments.

```suggestion
                // If variation has only one groupable attribute, it's key value will be main variation id.
                // For this reason $attributeData will not contain an attribute id and attribute value id.
```